python/jiminy_py/src/jiminy_py/log.py

```diff
@@ -217,9 +231,67 @@ def build_robot_from_log(
 
     return robot
 
+def build_robots_from_log(
+        log_data: Dict[str, Any],
+        mesh_path_dir: Optional[str] = None,
+        mesh_package_dirs: Sequence[str] = (),
+        ) -> Sequence[jiminy.Robot]:
+    """Build robots from log. For mulit-robot simulations, build all the robots
+    in the log of the simulation.
+
+    .. note::
```

Comment:
Remove redundant notes and refer to `build_robot_from_log` for more information.